### PR TITLE
Closes #365: Added Seed Script

### LINKED ISSUE #365: Add seed script
Add script to seed database with test data.

### PR DESCRIPTION
**Pull request by katconnors**
Seed script has been added to populate tables with test data.
One can use npm run db:seed to add three students and one staff member to the database.

image
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,12 @@ There are two ways to run Compass locally:
 
    NOTE: If you get into a login loop, double check that the Docker services are running
 
+4. Seed database (Optional)
+   ````sh
+   npm run db:seed           # Seed the database with test data
+   ```                       # Must be done after login to Compass app
```
[the remark this answers is not in the document]
Comment (mrabbitt) [reply README.md:63]: Thanks for documenting that the command must be run after logging into Compass to create the admin user. 👍

**Review by mrabbitt — COMMENTED** [review]
I tried out the `db:seed` command on my local environment and it worked great! I also tried out a couple of error scenarios:

1. If `db:seed` is run right after `db:reset` without logging in to create an admin user first, it failed as I'd expect. The error message is basically `NoResultError: no result` with a stacktrace. ✅ 
2. If `db:seed` is run a second time after it runs successfully, it fails as expected with `error: duplicate key value violates unique constraint "student_email_key"`. ✅ 

One other thought for future improvement: Should there be a test to make sure the `db:seed` command doesn't fail in the future if there are schema changes to the database?

I pointed out one typo and made a minor improvement suggestion in other comments, but otherwise it looks great to me. 

Thanks for adding this!

File: src/backend/db/lib/seed.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      email: "elastic@example.com",
+      role: "staff",
+    })
+    .execute();
+
+  const newstaff = await db
+    .selectFrom("user")
+    .select("user_id")
+    .where("first_name", "=", "Helen")
+    .executeTakeFirstOrThrow();
```
[the remark this answers is not in the document]
Comment (katconnors) [reply src/backend/db/lib/seed.ts]: Good callout. Updated this!

**Comment by katconnors** [comment]
Thanks for the feedback, @mrabbitt!

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (katconnors) [reply README.md]: You're welcome! Good suggestion- I'll open up a ticket for a test to be added.

**Review by mrabbitt — APPROVED** [review]
Looks good to me.

**Pull request merged**